packages/amplify-cli/src/index.ts

```diff
@@ -111,6 +115,13 @@ export async function run() {
   }
 }
 
+function verifyFilePermissions(filePath) {
```

Comment:
`ensureFilePermissions` probably a better name since we also changing it, not just verifying.